draft-ietf-taps-arch.md

```diff
@@ -553,14 +556,14 @@ the Encapsulating Security Protocol (ESP, part of IPsec) {{?RFC4303}} with QUIC,
 and applications ought not to use private keys intended for server
 authentication as keys for client authentication.
 
-Moreover, Transport Services systems must not automatically fall back from
+A Transport System implementing the Transport Services Architecture must not automatically fall back from
 secure protocols to insecure protocols, or to weaker versions of secure
 protocols (see {{equivalence}}). For example, if an application requests a specific version of TLS,
 but the desired version of TLS is not available, its connection will fail.
 Applications are thus responsible for implementing security protocol fallback
-or version fallback by creating multiple Transport Services Connections, if so
-desired. Alternatively, a Transport Services system MAY allow applications to
-specify that fallback to a specific other version of a protocol is allowed.
+or version fallback by creating multiple Connections, if so
+desired. Alternatively, the Transport Services API MAY allow applications to
+specify that the Transport System is allowed to fallback to a specific other version of a protocol.
```

Comment:
in the replaced text, "fallback" is used as a noun, but here it is a verb - I'm not sure it's a verb that exists? Maybe "falling back" ?